hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/BucketEndpoint.java

```diff
@@ -112,6 +112,9 @@ public Response list(
     ContinueToken decodedToken =
         ContinueToken.decodeFromString(continueToken);
 
+    if (startAfter == null) {
```

Comment:
We need some comments here.

Reply:
Done.

Reply:
What do you mean "stand for startAfter" ?